cluster/images/base.cmds

```diff
@@ -1,20 +1,20 @@
 # Eliminate stale cache issues with gsutil
-echo 'tmpfs /root/.gsutil tmpfs size=128m 0 0' | sudo tee -a /etc/fstab
-
-sudo apt -y upgrade
-sudo apt -y update
-sudo apt -y install dirmngr apt-transport-https ca-certificates software-properties-common gnupg2
-sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-key '95C0FAF38DB3CCAD0C080A7BDC78B2DDEABC47B7'
-sudo add-apt-repository 'deb https://cloud.r-project.org/bin/linux/debian stretch-cran35/'
-sudo apt -y update
-sudo apt -y install perl-modules make time python3-pip python3-venv r-base vcftools openjdk-8-jdk bc
-sudo apt -y -t stretch-backports install openjdk-11-jdk
-sudo apt -y install libcurl4-gnutls-dev libxml2-dev libssl-dev less libgd-dev cpanminus parallel pigz git libmagick++-dev
-sudo apt -y install mdadm --no-install-recommends
-sudo pip3 install --no-cache-dir -U crcmod
-sudo mkdir /data
-sudo mkdir /opt/tools
-sudo mkdir /opt/resources
+#echo 'tmpfs /root/.gsutil tmpfs size=128m 0 0' | sudo tee -a /etc/fstab
```

Comment:
I think the commenting will end up staying here and confusing someone down the line. Maybe write something as to why it was commented? Or put in a README?

Reply:
Agreed, as I was pushing it this way I was thinking exactly these thoughts. I'll comment probably just to leave a breadcrumb back to the repo in `master` and then maybe backport to the other active branches.